packit_service/service/api/koji_builds.py

```diff
@@ -65,22 +81,44 @@ def get(self):
 class KojiBuildItem(Resource):
     @koji_builds_ns.response(HTTPStatus.OK, "OK, koji build details follow")
     @koji_builds_ns.response(
-        HTTPStatus.NO_CONTENT.value, "Koji build identifier not in db/hash"
+        HTTPStatus.NOT_FOUND.value, "No info about build stored in DB"
     )
     def get(self, id):
         """A specific koji build details. From koji_build hash, filled by worker."""
         builds_list = KojiBuildModel.get_all_by_build_id(str(id))
         if bool(builds_list.first()):
```

Comment:
Could you take the "early return" approach here? Return early with `NOT_FOUND` if there is nothing in `builds_list`.

And maybe the `bool()` is not necessary.

(I know this is not strictly related to this PR, but if you touch this part anyways, let's fix it :slightly_smiling_face:)

Reply:
Sure